packs/data/heritages.db/luminous-sprite.json

```
@@ -6,9 +6,36 @@
             "uuid": "Compendium.pf2e.ancestries.TRqoeYfGAFjQbviF"
         },
         "description": {
-            "value": "<p>You glow with magical luminance. You naturally shed light as a torch; this light has the evocation, light, and primal traits. You can extinguish, rekindle, or change the color of this light using a single action, which has the concentrate trait.</p>"
+            "value": "<p>You glow with magical luminance. You naturally shed light as a @Compendium[pf2e.equipment-srd.Torch]{Torch}; this light has the evocation, light, and primal traits. You can extinguish, rekindle, or change the color of this light using a single action, which has the concentrate trait.</p>"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Sprite.LuminousSprite.Light",
+                "option": "luminous",
+                "toggleable": true
+            },
+            {
+                "key": "TokenLight",
+                "predicate": {
+                    "all": [
+                        "luminous"
+                    ]
+                },
+                "value": {
+                    "animation": {
+                        "intensity": 4,
+                        "speed": 1,
+                        "type": "torch"
+                    },
+                    "bright": 20,
+                    "color": "#9b7337",
+                    "dim": 40,
+                    "shadows": 0.2
+                }
```

```suggestion
                    "dim": 40,
                    "shadows": 0.2,
                    "luminosity": 0.4
                }
```
Toning it down just a hair.